### ISSUE TITLE
(asr.sh:252:main) ./asr.sh --stage 2 --ngpu 1 --train_set train --valid_set valid --test_sets test --lm_train_text data/train/text ./asr.sh: line 255: utils/parse_options.sh: No such file or directory

### ISSUE THREAD

**Issue by elhamadjidi** [espnet/espnet#4391]
I try to use my own dataset, with my own voice to train a model, I followed the instruction in [https://github.com/espnet/espnet/tree/master/egs2/TEMPLATE](url) but when I run this command `!./asr.sh \
 --stage 2 \
 --ngpu 1 \
 --train_set train \
 --valid_set valid \
 --test_sets "test" \
 --lm_train_text "data/train/text"` it gives me this error:
(asr.sh:252:main) ./asr.sh --stage 2 --ngpu 1 --train_set train --valid_set valid --test_sets test --lm_train_text data/train/text
./asr.sh: line 255: utils/parse_options.sh: No such file or directory

I am running the code on google colab, python version 3.7.13.

**Comment by ftshijt** [comment]
Could you check the following:
1. have you installed the espnet following the installation guide (which will clone kaldi for the corresponding file)?
2. where do you execute the commend? That should be in folders like `egs2/foo/${task}`

The `parse_options.sh` should be in `egs2/foo/${task}` piror to your running of asr.sh

**Comment by elhamadjidi** [comment]
Yes I solved that, Now it gives me the error of `line 309: local/data.sh: No such file or directory` I am trying to fine-tune on of pre-trained model with my own dataset. Would you tell me how I can make the data.sh?

**Comment by ftshijt** [comment]
Please check the guidelines here https://github.com/espnet/espnet/tree/master/egs2/TEMPLATE#for-developers-how-to-makeport-new-recipe You can find various examples of `data.sh` at https://github.com/espnet/espnet/tree/master/egs2

**Comment by elhamadjidi** [comment]
I managed to do the data partitioning with python and started `tts.sh` from stage 2, is that okay? I really didn't want to deal with `data.sh`

**Comment by ftshijt** [comment]
The introduction of data.sh is only to uniform our recipe layout. If you have prepared the data already, there is no need for the step to train a model (and do the inference, and scoring). But if you are thinking about contributing a recipe, please add that.
